packages/tools/viewer-configurator/src/hooks/observableHooks.ts

```diff
@@ -10,6 +10,7 @@ import { useEffect, useState } from "react";
  * @param eventNames The names of the events to listen for.
  * @returns The current value of the accessor.
  */
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export function useEventfulState<T>(accessor: () => T, element: HTMLElement | null | undefined, ...eventNames: string[]): T {

```

Comment:
These functions are really only intended for internal use within the Configurator, so I would think it would be fine to update the casing.

Reply:
This is a specific case (which I will be happy to fix), but in general, any exported function was left as is unless marked `@internal`

Reply:
This is an interesting case, because React already has standard naming conventions for hooks. In this case, I feel like it would make more sense to align with React's naming conventions than the Babylon API naming conventions. It's fuzzy though and I could maybe see arguments both ways.

Reply:
I 100% see your point. For react developers it would be weird using "`UseEventfulState`". This can always be added as exception (even for specific files). I am going to resolve and merge this later, but I will wait for your final OK before doing so.

Reply:
Resolving for now, can be discussed